src/nav-bar/announcement-banner.md

```diff
@@ -0,0 +1,6 @@
+---
+template: announcement
+title: Announcement
+---
+
+Find out what's next... <a href="https://blog.threshold.network/" target="_blank" rel="noopener noreferrer">Learn More</a>
```

Comment:
> What if we fetched all the text content such as Find out what's next..., Learn More, etc and then we could use them inside of the JSX.

As I mentioned here https://github.com/threshold-network/splash-page/pull/16#discussion_r827792831 if we will agree on the template for a given part of the UI eg. announcement banner it's OK. But I think injecting markdown via `dangerouslySetInnerHTML={{ __html: body }}` gives us more power because we can put into the announcement banner 2 external links via cms instead of a fixed template of a given component. eg here:
```
<Alert
      bg="brand.500"
      color="white"
      justifyContent="center"
      position="relative"
    >
      {body.linkTextPrefix}
      <ExternalLink
        containerProps={{ ml: "5px" }}
        href={ExternalLinkHref.THRESHOLD_BLOG}
      >
          {body.linkText}
      </ExternalLink>
      <CloseButton position="absolute" right="10px" onClick={onClose} />
    </Alert>
```
you can't add another external link at the end of the banner via cms because a component has a fixed template.